sunpy/time/time.py

```diff
@@ -34,13 +34,27 @@
     '%f': r'(?P<microsecond>\d+)',
     '%b': r'(?P<month_str>[a-zA-Z]+)',
 }
+# DO NOT SORT THIS LIST.
+# The string parsing is dependent on the specific order within this list.
 TIME_FORMAT_LIST = [
+    """
```

Comment:
Why was this added to the list?

Reply:
Also could this not be a comment with the format it affects?

Reply:
I thought this would be best way to explain,
now i have wrote a more concise explanation